lib/AnyDBM_File.t

```diff
@@ -1,18 +1,20 @@
 #!./perl
 
+use strict;
+use warnings;
+
 BEGIN {
     chdir 't' if -d 't';
     @INC = '../lib';
-    require Config; import Config;
-    require Test::More; import Test::More;
-    plan(tests, 12);
+    use Config;
```

Comment:
This looks wrong to me. The `@INC` manipulation is intended to affect the `use Config` and `use Test::More`, and this way I think it wouldn't

Reply:
Might be best to just fix the indirect imports each time this appears to keep changes minimal?

Reply:
AFAIK This is standard boiler plate to make sure the starting dir is right for the test. Config looks like it can come out. Andy removed the use of it in be6bd645f6be4d28e5d344c51889c6b67bdf5a74